core/trino-main/src/test/java/io/trino/sql/planner/iterative/rule/TestApplyTableScanRedirection.java

```diff
@@ -189,7 +192,11 @@ public void testApplyTableScanRedirection()
                             tableScan(
                                     equalTo(new MockConnectorTableHandle(destinationTable)),
                                     TupleDomain.all(),
-                                    ImmutableMap.of("DEST_COL", equalTo(destinationColumnHandleA))));
+                                    ImmutableMap.of("DEST_COL", equalTo(destinationColumnHandleA))))
+                    .producesWarnings(
+                            ImmutableList.of(new TrinoWarning(
```

Comment:
I think `.producesWarnings` could be varargs

Reply:
changed to varargs